release-1.6/release-notes-draft.md

```diff
@@ -53,8 +53,49 @@
 ### New Features
 - Add new command "kubectl set selector" ([#38966](https://github.com/kubernetes/kubernetes/pull/38966), [@kargakis](https://github.com/kargakis))
 - Add kubectl create poddisruptionbudget command ([#36646](https://github.com/kubernetes/kubernetes/pull/36646), [@kargakis](https://github.com/kargakis))
+- Add new DaemonSet status fields to kubectl printer and describer.  ([#42843](https://github.com/kubernetes/kubernetes/pull/42843), [@janetkuo](https://github.com/janetkuo))
```

Comment:
There are no new features for SIG-CLI in 1.6 release per the features tracking [spreadsheet](https://docs.google.com/spreadsheets/d/1nspIeRVNjAQHRslHQD1-6gPv99OcYZLMezrBe3Pfhhg/edit#gid=0).

Please, move the notes from "new features" section to "notable changes" or "breaking changes" sections.